### PR TITLE
Closes #6632: Image Optimization Tab Update

### LINKED ISSUE #6632: Image Optimization Tab Update
We would like to update the content for the Image Optimization Tab in the WP Rocket plugin. 

Current content:
image

New content for users who don't have Imagify:
image

New content for users who have Imagify plugin installed but not activated:
image

👉 [Mocflow doc](https://wireframepro.mockflow.com/editor.jsp?editor=off&perm=Owner&projectid=MtikZt9NJh&publicid=f7eae94ccd71483c96e30142ef52864f&space=default#/page/De096a4d4ec5f8943a2493ebc18f5e3be/mode/design)

### PR DESCRIPTION
**Pull request by jeawhanlee**
# Description
Updates the UI of the Image Optimization tab.

Fixes #6632 

### User documentation

*Explain how this code impacts users.*

### Technical documentation

*Explain how this code works. Diagram & drawings are welcomed.*

## Type of change

- [x] Enhancement (non-breaking change which improves an existing functionality).

## New dependencies
`wp-admin/includes/plugin-install.php`

## Risks
None

# Checklists

## Feature validation

- [ ] I validated all the Acceptance Criteria. If possible, provide sreenshots or videos.
- [ ] I triggered all changed lines of code at least once without new errors/warnings/notices.
- [ ] I implemented built-in tests to cover the new/changed code.

## Documentation

- [ ] I prepared the user documentation for the feature/enhancement and shared it in the PR or the GitHub issue.
- [ ] The user documentation covers new/changed entry points (endpoints, WP hooks, configuration files, ...).
- [ ] I prepared the technical documentation if needed, and shared it in the PR or the GitHub issue.

## Code style
- [x] I wrote self-explanatory code about what it does.
- [ ] I wrote comments to explain why it does it.
- [x] I named variables and functions explicitely.
- [ ] I protected entry points against unexpected inputs.
- [x] I did not introduce unecessary complexity.
- [x] I listed the introduced external dependencies explicitely on the PR.
- [ ] I validated the repo-specific guidelines from CONTRIBUTING.md.

## Observability
- [x] I handled errors when needed.
- [ ] I wrote user-facing messages that are understandable and provide actionable feedbacks.
- [ ] I prepared ways to observe the implemented system (logs, data, etc.).

## Risks
- [ ] I explicitely mentioned performance risks in the PR.
- [ ] I explicitely mentioned security risks in the PR.

### CODE REVIEW

**Comment by jeawhanlee** [comment]
### State without Imagify installed.
Screenshot 2024-05-28 at 18 40 10

**Comment by jeawhanlee** [comment]
### State with Imagify Installed but not activated.
Screenshot 2024-05-28 at 18 42 16

**Comment by jeawhanlee** [comment]
### State with Imagify Installed & Activated.
Screenshot 2024-05-28 at 18 44 19

**Review by Miraeld — APPROVED** [review]

**Comment by marija-nikolic** [comment]
> ### State with Imagify Installed & Activated.
> Screenshot 2024-05-28 at 18 44 19

We don't have an Image Optimization tab at all when Imagify is Installed and Activated. That is currently the case.

File: inc/Engine/Admin/Settings/Render.php [hunk 1]
```diff
@@ -130,7 +130,53 @@ public function render_form_sections() {
...
+				],
+			];
+
+			$plugin_data = plugins_api( 'plugin_information', $query_args );
+
+			if ( is_wp_error( $plugin_data ) ) {
+				$plugin_data = [];
+			}
+
+			set_transient( 'imagify_plugin_data', $plugin_data, WEEK_IN_SECONDS );
```
Comment (remyperona) [inline inc/Engine/Admin/Settings/Render.php]:
```suggestion
			set_transient( 'rocket_imagify_plugin_data', $plugin_data, WEEK_IN_SECONDS );
```

File: views/settings/page-sections/imagify.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /**
  * Imagify section template.
  *
- * @since 3.2
+ * @since 3.2ƒ
```
Comment (remyperona) [inline views/settings/page-sections/imagify.php]: is this a typo?

**Review by remyperona — CHANGES_REQUESTED** [review]

File: views/settings/page-sections/imagify.php (hunk 2 above)
Comment (jeawhanlee) [reply views/settings/page-sections/imagify.php]: Definitely a typo.

**Review by remyperona — APPROVED** [review]

**Comment by remyperona** [comment]
Display is good 👍🏼

**Pull request merged**